app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(:user_id, :id)
+    if @like.save
```

Comment:
- [ ] The functionality to like a post is not working as expected. Currently clicking on the button simply refreshes the page and does not actually like the post. Please try to fix this 🙏 
**HINT:**  The `postid` is unaccounted for so which post are you trying to add a like to?👍 
![image](https://user-images.githubusercontent.com/46839548/215005484-a4f63c1d-6515-4c64-b8b6-5f26b96da84c.png)

Reply:
hi @codepantha  im trying to add a like for the comment that is for the post, but I could deploy it, could you please see what is the error ??